test/runtests.jl

```diff
@@ -212,6 +212,39 @@ end
     @test isapprox( signalsum, T(2), atol = 5e-3 )
 end
 
+@testset "Isochrone" begin
+    sim = Simulation{T}(SSD_examples[:IsochroneTest])
+    simulate!(sim, device_array_type = device_array_type)
+    
+    calculate_electric_potential!( sim, refinement_limits = [0.2, 0.1, 0.05, 0.01])
+    calculate_electric_field!(sim, n_points_in_φ = 72)
+    charge_drift_model = ADLChargeDriftModel()
+    sim.detector = SolidStateDetector(sim.detector, charge_drift_model);
+
+    spawn_positions = CartesianPoint{T}[]
+    idx_spawn_positions = []
+    x_axis = sim.electric_field.grid.axes[1][1:2:end]
+    z_axis = sim.electric_field.grid.axes[3][1:2:end]
+    for (i,x) in enumerate(x_axis)
+        for (k,z) in enumerate(z_axis)
+            push!(spawn_positions, CartesianPoint([x,0,z]))
+            push!(idx_spawn_positions, CartesianIndex(i,k))
+        end
+    end
+    length(spawn_positions)
+    in_idx = findall(x -> x in sim.detector && !in(x, sim.detector.contacts), spawn_positions);
+    ev = Event(spawn_positions[in_idx]);
+    time_step = 0.8u"ns"
+    drift_charges!(ev, sim, Δt = time_step, verbose = false)
+
+    DT = fill(NaN,length(x_axis),length(z_axis))
+    for (i, idx) in enumerate(idx_spawn_positions[in_idx])
+        DT[idx] = length(ev.drift_paths[i].h_path)*ustrip(time_step)
+    end
+    @test maximum(DT[35:50,60:80])<100
```

Comment:
If you only check part of `DT`, then you should only drift charges for these points
(not the full `x_axis` and `z_axis` but only the part you check)